concepticondata/conceptlists/Sidwell-2021-100.tsv

```diff
@@ -0,0 +1,101 @@
+ID	NUMBER	ENGLISH	SWADESH_1952_200	SWADESH_1955_100	CONCEPTICON_ID	CONCEPTICON_GLOSS
+Sidwell-2021-100-1	1	Belly/stomach	10	4	2114	BELLY OR STOMACH
+Sidwell-2021-100-2	2	Big/large	11	5	1202	BIG
+Sidwell-2021-100-3	3	Bird	12	6	937	BIRD
+Sidwell-2021-100-4	4	Bite(v.)		7	1403	BITE
+Sidwell-2021-100-5	5	Black	14	8	163	BLACK
+Sidwell-2021-100-6	6	Blood	15	9	946	BLOOD
+Sidwell-2021-100-7	7	Bone		10	1394	BONE
+Sidwell-2021-100-8	8	Breast		11	1402	BREAST
+Sidwell-2021-100-9	9	Burn(intr.)		12	2102	BURN
```

Comment:
This entry should probably me mapped to `BURNING`, as it is explicitly intransitive

Reply:
Nice catch.

Reply:
When recovering the deleted data, you also restored the old mappings. Please map to `BURNING` again.